packages/front-end/components/Features/EditFeatureInfoModal.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC } from "react";
+import { useForm } from "react-hook-form";
+import { FeatureInterface } from "back-end/types/feature";
+import { Box } from "@radix-ui/themes";
+import Modal from "@/components/Modal";
+import Field from "@/components/Forms/Field";
+import TagsInput from "@/components/Tags/TagsInput";
+import SelectOwner from "@/components/Owner/SelectOwner";
+import useProjectOptions from "@/hooks/useProjectOptions";
+import SelectField from "@/components/Forms/SelectField";
+import Callout from "@/components/Radix/Callout";
+import usePermissionsUtil from "@/hooks/usePermissionsUtils";
+import Tooltip from "@/components/Tooltip/Tooltip";
+
+const EditFeatureInfoModal: FC<{
+  feature: FeatureInterface;
+  save: (updates: {
+    tags: string[];
+    owner: string;
+    project: string;
+  }) => Promise<void>;
+  cancel: () => void;
+  mutate: () => void;
+  source?: string;
+  resourceType: React.ComponentProps<typeof SelectOwner>["resourceType"];
+  dependents: number;
+}> = ({ feature, save, cancel, mutate, source, resourceType, dependents }) => {
+  const form = useForm({
+    defaultValues: {
+      tags: feature.tags || [],
+      owner: feature.owner,
+      project: feature.project || "",
+    },
+  });
+  const permissionsUtil = usePermissionsUtil();
+
+  const permissionRequired = (project) =>
+    permissionsUtil.canUpdateFeature({ project }, {});
```

Comment:
This isn't quite right.  It should be `canUpdateFeature(feature, {project})`.  We want to make sure they have permission to edit in both the original and new project.

Reply:
Fixed